crates/sui-framework/sources/governance/voting_power.move

```diff
@@ -142,6 +143,7 @@ module sui::voting_power {
         let total = 0;
         while (i < len) {
             let voting_power = validator::voting_power(vector::borrow(v, i));
+            assert!(voting_power > 0, EInvalidVotingPower);
```

Comment:
is this voting power > 0 or should be at least minimum voting power >=0.25?

Reply:
This is normalized voting_power (i.e. when total voting power is constant 10000). Even if the minimum stake is 0.2% of total supply, there is no guarantee that the minimum voting power is 0.2%.

Reply:
there should be some guarantees though no? Can everyone have 0.01% and all o them do not surpass 67%? Can someone have more than 10% of total voting power?

Reply:
> Can everyone have 0.01%

Assuming you means "someone" instead of "everyone", yes I think so. There is nothing preventing this.

> Can someone have more than 10% of total voting power?

Yes, when there are only 4 validators, each validator would have 25% voting power.

Reply:
Ok iiuc it's a bit complicated to define some limits (although we could add some restrictions) ie no less than 4 validators and thus max 25%. I'm ok without adding these rules because we indeed to always allow progress, unless there are situations where progress won't be permitted due to breaking our decentralisation assumptions.